shared/actions/chat/inbox.js

```diff
@@ -41,9 +42,14 @@ function * onInitialInboxLoad (): SagaGenerator<any, any> {
   if (inboxUntrustedState === 'unloaded' || _inboxUntrustedError) {
     yield put(Creators.setInboxUntrustedState('loading'))
     _inboxUntrustedError = null
-    yield call(onInboxStale)
```

Comment:
We have this pattern in a couple of places where we'll explicitly call some helper which also happens to be takeLatest() on. This means that those calls and this one actually can run in parallel which we likely dont' want. Instead of us doing this lets just dispatch the action and let the normal mechanism handle it. This also means we reduce the variants of the flows we have which likely makes this easier to reason about